torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -117,34 +101,6 @@ struct ConstantPropagator {
     return var_outputs;
   }
 
-  // Tuples are not representable as constants, however
```

Comment:
This sounds like since tuples are representable as constants they wouldn't need to be handled here, but that doesn't happen until the PR above this one in the stack, so how does this still work?

Reply:
i'm going to land the stack in one.